recipes/default.rb

```diff
@@ -23,182 +23,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# workaround to allow for a nagios server install from source using the override attribute on debian/ubuntu (COOK-2350)
-nagios_service_name = if platform_family?('debian') && node['nagios']['server']['install_method'] == 'source'
-                        node['nagios']['server']['name']
-                      else
-                        node['nagios']['server']['service_name']
-                      end
-
-# install nagios service either from source of package
-include_recipe "nagios::server_#{node['nagios']['server']['install_method']}"
-
 # configure either Apache2 or NGINX
 case node['nagios']['server']['web_server']
 when 'nginx'
   Chef::Log.info 'Setting up Nagios server via NGINX'
   include_recipe 'nagios::nginx'
-  web_user = node['nginx']['user']
-  web_group = node['nginx']['group'] || web_user
 when 'apache'
   Chef::Log.info 'Setting up Nagios server via Apache2'
   include_recipe 'nagios::apache'
-  web_user = node['apache']['user']
-  web_group = node['apache']['group'] || web_user
 else
-  Chef::Log.fatal('Unknown web server option provided for Nagios server: ' \
+  Chef::Log.info('Unknown web server option provided for Nagios server: ' \
```

Comment:
I wanted to allow the possibility of using the nagios cookbook without configuring a web server, which is why there is an  "include_recipe 'nagios::server'" line below the info.

I changed the logic a bit and added the option 'none' to make it more explicit.